components/accordion/index.css

```diff
@@ -218,124 +260,78 @@
+	padding-inline-end: var(--mod-accordion-edge-to-text-space, var(--spectrum-accordion-edge-to-text-space));
+}
+
+/* Focusable button that expands/collapses the accordion item. */
 .spectrum-Accordion-itemHeader {
+	box-sizing: border-box;
 	position: relative;
+
 	display: flex;
-	align-items: center;
+	align-items: flex-start;
 	justify-content: flex-start;
 
-	box-sizing: border-box;
-	padding-block: var(--mod-accordion-item-header-top-to-text-space, var(--spectrum-accordion-item-header-top-to-text-space)) var(--mod-accordion-item-header-bottom-to-text-space, var(--spectrum-accordion-item-header-bottom-to-text-space));
-
-	padding-inline-start: calc(var(--mod-accordion-disclosure-indicator-to-text-space, var(--spectrum-accordion-disclosure-indicator-to-text-space)) + var(--mod-accordion-disclosure-indicator-height, var(--spectrum-accordion-disclosure-indicator-height)));
-	padding-inline-end: var(--mod-accordion-edge-to-text-space, var(--spectrum-accordion-edge-to-text-space));
-
-	min-block-size: var(--mod-accordion-min-block-size, var(--spectrum-accordion-min-block-size));
+	/* start spacing controlled by edge to disclosure icon spacing */
+	padding-inline: 0 var(--mod-accordion-edge-to-content-area, var(--spectrum-accordion-edge-to-content-area));
 	line-height: var(--mod-accordion-item-header-line-height, var(--spectrum-accordion-item-header-line-height));
 
 	text-overflow: ellipsis;
-	cursor: pointer;
+	cursor: var(--spectrum-accordion-item-header-cursor);
 	font-size: var(--mod-accordion-item-header-font-size, var(--spectrum-accordion-item-header-font-size));
 	font-weight: var(--mod-accordion-item-header-font-weight, var(--spectrum-accordion-item-header-font-weight));
 	font-style: var(--mod-accordion-item-header-font-style, var(--spectrum-accordion-item-header-font-style));
 	font-family: var(--mod-accordion-item-header-font, var(--spectrum-accordion-item-header-font));
 
-	/* reset styling if button element is used */
+	/* Reset styling of button element */
 	appearance: none;
 	border: 0;
 	text-align: start;
 	inline-size: 100%;
 
-	&:focus {
-		outline: none;
-
-		&::after {
-			content: "";
-			position: absolute;
-			inset-inline-start: 0;
-		}
-	}
 	color: var(--mod-accordion-item-header-color-default, var(--spectrum-accordion-item-header-color-default));
 	background-color: var(--mod-accordion-background-color-default, var(--spectrum-accordion-background-color-default));
 
-	&:hover {
+	&:hover &:not(.is-disabled) {
 		color: var(--mod-accordion-item-header-color-hover, var(--spectrum-accordion-item-header-color-hover));
 		background-color: var(--mod-accordion-background-color-hover, var(--spectrum-accordion-background-color-hover));
-
-		+ .spectrum-Accordion-itemIconContainer {
-			color: var(--mod-accordion-item-header-color-hover, var(--spectrum-accordion-item-header-color-hover));
-		}
 	}
 
-	&:focus-visible {
+	&:focus-visible &:not(.is-disabled) {
```

Comment:
I'm curious if we need the space in this selector. With the space between `&:focus-visible` and `&:not(.is-disabled)`, I don't actually see any of the accordion interactive states (hover, focus, active, active+hovered, etc). 

If I remove the space, I see the styles. If I remove the space and the next ampersand, I see the styles. I _don't_ however, know which of those options are the right fix.

Reply:
Actually, looking at the token specs more, it almost looks like the focus ring applies to the `.spectrum-Accordion-item`, as opposed to `.spectrum-Accordion-itemHeader` (which is the button element and probably what is _actually_ getting the focus). Josh and/or SaaSha may have already asked if it's ok that our border-block-end peeks through behind the focus ring.

Figma (the divider gets cut off by the focus ring)
<img width="339" alt="Screenshot 2025-06-11 at 5 33 11 PM" src="https://github.com/user-attachments/assets/36f31d2d-e5a2-451f-8a96-aebb210e2502" />

Ours (the divider shows behind the focus ring)
<img width="143" alt="Screenshot 2025-06-11 at 5 32 22 PM" src="https://github.com/user-attachments/assets/5728a89c-0a3a-44a3-9238-b08cd19668e7" />

Reply:
I left another comment about it but I'm not convinced it's _just_ the gap that would need some attention. 

This screenshot has the 2px gap on that bottom focused item, and that makes the overall width of that item too large. 

<img width="514" alt="Screenshot 2025-06-13 at 3 02 26 PM" src="https://github.com/user-attachments/assets/c0212e50-8b9c-4435-bd24-9e35d8c768db" />